e2e/specs/publish-form.spec.ts

```diff
@@ -23,6 +23,10 @@ test("Publish a form", async ({ page }) => {
     await formBuilderPage.gotoFormBuilder();
   });
 
+  await test.step("And I search for the form I need to publish", async () => {
+    await formBuilderPage.searchForm(form.name);
+  });
+
   await test.step("And I click on a form I need to publish", async () => {
     await page.getByTestId(`editSchema${form.uuid}`).click();
```

Comment:
@Piumal1999 Now we don't need to use the form uuid to edit right? Because in the UI there will be only the relevant form. So we can just click the edit button.

Reply:
Yeah. Is it okay if we use the following selector?
```
getByRole('row', { name: form.name }).getByRole('button').first()
```

I'm not sure whether using `.first()` is a good practice or not.

Reply:
can we use the `Edit schema` tooltip here?

Reply:
I think it's fine to use the `first` if someone changes the order, the PR would fail. So that they can update the test. 

What do you think?

Reply:
yeah, we can, but using a visible label or a tooltip would be ideal because we are mimicking a user here. @Piumal1999 can you try it? If not let's move forward with the `first` approach.

Reply:
@Piumal1999 were you able to try Anjula's suggestion?

Reply:
> @Piumal1999 were you able to try Anjula's suggestion?

Oh I had pushed the wrong commit instead of the updated one.

Reply:
Fixed with [5ca6640](https://github.com/openmrs/openmrs-esm-form-builder/pull/157/commits/5ca6640fdf15d1b3a18b74159b1f9741cec79dc0)